src/App.tsx

```diff
@@ -74,7 +79,7 @@ export const App = () => {
 
   return (
     <>
-      {showConsentBanner && <ConsentBanner />}
+      {showConsentBanner && isMixpanelEnabled && !isMobileApp && <ConsentBanner />}
```

Comment:
This was derp to keep at selector-level - this wasn't doing much in terms of selecting from state (we were actually being reactive on the *whole* app state there and doing a `selector(state)` within a selector).

Moved logic to view-layer i.e we consume `selectShowConsentBanner` directly from the slice (i.e `preferences.showConsentBanner`), and then we select the flag separately and do the `isMobileApp` checks inline here